### ISSUE TITLE
Problem with cssnano in package.json

### ISSUE THREAD

**Issue by DrizzlyOwl** [roikles/Flexbones#47]
`sudo npm install` was executed in a fresh project folder.
`npm set progress=false` was set to reduce terminal output during install.

`npm run watch` was executed once the install was complete. output is as follows:

```
➜  plex git:(dev) ✗ npm run watch

> Flexbones@3.0.0 watch /Applications/MAMP/htdocs/plex
> livereload & npm run watch:sass & npm run watch:js

Starting LiveReload v0.4.1 for /Applications/MAMP/htdocs/plex on port 35729.

> Flexbones@3.0.0 watch:sass /Applications/MAMP/htdocs/plex
> fsmonitor -d 'assets/sass' -s '+*.scss' npm run css


> Flexbones@3.0.0 watch:js /Applications/MAMP/htdocs/plex
> fsmonitor -d 'assets/js' -s '+*.js' npm run browserify -s



Monitoring:  assets/sass
Monitoring:  assets/js
    filter:  **/*.scss/**
    action:  npm run css

    filter:  **/*.js/**
    action:  npm run browserify -s

npm run css
npm run browserify -s
......
> Flexbones@3.0.0 css /Applications/MAMP/htdocs/plex
> npm run generate-css && npm run generate-admin-css


> Flexbones@3.0.0 generate-css /Applications/MAMP/htdocs/plex
> node-sass --output-style expanded assets/sass/style.scss style.css && pixrem -i style.css -o style.css -r 10px && postcss --use autoprefixer -o style.css style.css && cssnano style.css style.css --no-autoprefixer --no-convertValues


...Rendering Complete, saving .css file...
Wrote CSS to /Applications/MAMP/htdocs/plex/style.css
Pixrem complete.
sh: cssnano: command not found

npm ERR! Darwin 15.2.0
npm ERR! argv "/usr/local/Cellar/node/5.4.1_1/bin/node" "/usr/local/bin/npm" "run" "generate-css"
npm ERR! node v5.4.1
npm ERR! npm  v3.5.3
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! Flexbones@3.0.0 generate-css: `node-sass --output-style expanded assets/sass/style.scss style.css && pixrem -i style.css -o style.css -r 10px && postcss --use autoprefixer -o style.css style.css && cssnano style.css style.css --no-autoprefixer --no-convertValues`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the Flexbones@3.0.0 generate-css script 'node-sass --output-style expanded assets/sass/style.scss style.css && pixrem -i style.css -o style.css -r 10px && postcss --use autoprefixer -o style.css style.css && cssnano style.css style.css --no-autoprefixer --no-convertValues'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the Flexbones package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-sass --output-style expanded assets/sass/style.scss style.css && pixrem -i style.css -o style.css -r 10px && postcss --use autoprefixer -o style.css style.css && cssnano style.css style.css --no-autoprefixer --no-convertValues
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs Flexbones
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls Flexbones
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Applications/MAMP/htdocs/plex/npm-debug.log

npm ERR! Darwin 15.2.0
npm ERR! argv "/usr/local/Cellar/node/5.4.1_1/bin/node" "/usr/local/bin/npm" "run" "css"
npm ERR! node v5.4.1
npm ERR! npm  v3.5.3
npm ERR! code ELIFECYCLE
npm ERR! Flexbones@3.0.0 css: `npm run generate-css && npm run generate-admin-css`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the Flexbones@3.0.0 css script 'npm run generate-css && npm run generate-admin-css'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the Flexbones package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run generate-css && npm run generate-admin-css
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs Flexbones
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls Flexbones
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Applications/MAMP/htdocs/plex/npm-debug.log
```

tl;dr: error is:

```
sh: cssnano: command not found

npm ERR! Darwin 15.2.0
npm ERR! argv "/usr/local/Cellar/node/5.4.1_1/bin/node" "/usr/local/bin/npm" "run" "generate-css"
npm ERR! node v5.4.1
npm ERR! npm  v3.5.3
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn
npm ERR! Flexbones@3.0.0 generate-css: `node-sass --output-style expanded assets/sass/style.scss style.css && pixrem -i style.css -o style.css -r 10px && postcss --use autoprefixer -o style.css style.css && cssnano style.css style.css --no-autoprefixer --no-convertValues`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the Flexbones@3.0.0 generate-css script 'node-sass --output-style expanded assets/sass/style.scss style.css && pixrem -i style.css -o style.css -r 10px && postcss --use autoprefixer -o style.css style.css && cssnano style.css style.css --no-autoprefixer --no-convertValues'.
```

**Comment by roikles** [comment]
Fixed!!
